tests/test_sentry.py

```diff
@@ -148,13 +148,33 @@ def test_log_client(monkeypatch, log):
 
 def test_get_middlware():
     mw = talisker.sentry.get_middleware(lambda: None)
-    assert isinstance(mw, raven.middleware.Sentry)
+    assert isinstance(mw, talisker.sentry.TaliskerSentryMiddleware)
     assert mw.client == talisker.sentry.get_client()
     updates = talisker.sentry.sentry_globals['updates']
     assert len(updates) == 1
     assert updates[0].__closure__[0].cell_contents == mw
 
 
+def test_middleware_clears_context(environ):
+    logger = logging.getLogger(__name__)
+
+    def app(environ, sr):
+        logger.info('app log')
+        return []
+
+    mw = talisker.sentry.get_middleware(app)
+    context = mw.client.context
+    context.clear()  # clear startup logs
+    assert len(context.breadcrumbs.buffer) == 0
+    mw(environ, lambda: None)
```

Comment:
As the middleware itself should clear the contents I don't think l.167 is really required, moreover IIUC two or more calls of this kind (`mw(environ, lambda: None)`) should result in the assertions below still as still passing as the context shall be cleared every-time, maybe it'd be a good idea to just test this that way.